lib/tokenize.cpp

```diff
@@ -9093,6 +9093,11 @@ void Tokenizer::simplifyNamespaceStd()
 
     for (const Token* tok = Token::findsimplematch(list.front(), "using namespace std ;"); tok; tok = tok->next()) {
         bool insert = false;
+        if (Token::Match(tok, "enum class|struct| %name%| :|{")) { // Don't replace within enum definitions
+            const Token *defStart = Token::findsimplematch(tok, "{");
```

Comment:
This would not work well for garbage code

    enum E : int; ...

It would be more robust with:

    defStart = Token::findmatch(tok, "[:{]");
    while (Token::Match(defStart, "%name%|::|:"))
        defStart = defStart->next();
    if (defStart && defStart->str() == "{")
        tok = defStart->link();